### ISSUE TITLE
[BUG] - New token wallet get lost after creation

### ISSUE THREAD

**Issue by t-jacobi** [casinocoin/tokenwallet-mobile#154]
**Describe the bug**
If you create a new token wallet with an existing funded csc wallet it will be shown after creation. But if you switch to dashboard and back to wallet it will be gone. 

If you click in the Dashboard on the CSC Wallet you will switch to the wallet area without the token. if you tab then on wallet in the lower menu it will refresh the view with the token... 

**To Reproduce**
see above

**Expected behavior**

**Screenshots**

**Smartphone (please complete the following information):**
 - Device: iPhone6
 - OS: iOS
 - OS Version 13.3.1
 - App Version 0.1.14

**Additional context**

**Comment by t3nshi** [comment]
Not really sure I understand this that's why we ask for explicit steps. Your description is very good but not sure about what you are doing exactly. Let me rephrase it to confirm:

**To Reproduce**
Steps to reproduce the behavior:
1. Create a wallet with CSC activated and funded account.
2. Tap on add token on wallet page and add a token to the account and see the token account created
3. Tap on dashboard on the tab bar at the bottom
4. On dashboard tap on wallet on the tab bar at the bottom to return to the Wallet page.
5. Token account is gone, you only see the CSC account.
6a. After restarting the app  the token account is again on the wallet ???
6a. Even after restarting the app the token account is nowhere to be seen. ???

Since I am not sure what steps you took after many times I still can't reproduce this bug. Please let me know if you can provide additional information on this. thanks

**Comment by t-jacobi** [comment]
6a. If you tab on the wallet in the dashboard -> you wont see the token wallet
6a. if you tab on wallet on the buttom menu -> you will see the token wallet

**Comment by t3nshi** [comment]
What you just posted does not make any sense... do you mean that all the process explained is what you experience? and after step 5 you have step 6a producing two different results? I showed you how to make a detailed step by step so I can make the same process and have the same problem. But instead you are still refusing to provide steps, put ambiguous comments. I can help you like this. 
I hope that at least you copy and paste the process I drafted, correct the parts that are different and explain a bit. As I told you before, I make those steps and the token account don't dissapear for me. It does not mean it doesn't happen, but if I am still unable to properly understand your problem, to replicate it, or at least confirm what steps you took to get there so I know that the issue only happens on your device/setup... we will get nowhere and it is a waste of our time for both of us.

**Comment by t-jacobi** [comment]
Uploaded a video in a zip! If you go into the wallet from the dashboard view you will not see the token, when you click on the bottom "wallet" you will see the token... 

[RPReplay_Final1581194897.mp4.zip](https://github.com/casinocoin/tokenwallet-mobile/files/4175146/RPReplay_Final1581194897.mp4.zip)

**Comment by t3nshi** [comment]
I could not reproduce the video right now. I will try again on monday at the office. Let's hope that we can wrap it up.

**Comment by t3nshi** [comment]
LOL. I just watched the video. And I could see what you were talking about and now your comments make more sense... It is just, that tapping on any of the balances of the dashboard (CSC or PCN if you have any PCN balance, or other token) just brings the filtered accounts you have of that "currency" if you had any PCN accounts and you tap on the PCN balance, you will see the filtered wallet only showing those PCN accounts. It is a feature that was requested on  an earlier alpha of the app.
take a look at this small demo: 
[Screen Recording 2020-02-10 at 11.37.51-1.mov.zip](https://github.com/casinocoin/tokenwallet-mobile/files/4182010/Screen.Recording.2020-02-10.at.11.37.51-1.mov.zip)

**Comment by t3nshi** [comment]
I am closing this since it is not an issue. If you find that you still think there is a problem that needs fixing, reopen the issue or let us now with details so we reopen it.
